app/Http/Controllers/PostController.php

```diff
@@ -30,19 +29,18 @@ public function create()
     public function store()
     {
 
-      //die and dump
-       // dd(request()->all());
+      $this->validate(request(),[
+        'name' => 'required',
+        'email' => 'required',
+        'message' => 'required'
+      ]);
 
         // Create a new post using the request data
 
         Post::create(request(['name', 'email', 'message']));
 
         \Mail::to('jscher01@gmail.com')->send(new ContactUs);
 
-        //this next line may be better
-        //\Mail::to($email)->send(new ContactUs);
-
-        //????????should add something that said it worked.
         return redirect('/');
```

Comment:
Just a tip for the future, but you can redirect with a message so you can display a success message if it goes through :D

Reply:
```php
return redirect('/')->with(['message' => 'Message sent!']);
```

then you can access it in the controller with something like the following 

```blade
@if (isset($message) && !empty($message)) 
  <div class="alert alert-success">
      <b>{{ $message }}</b>
   </div>
@endif
```